README.md

```diff
@@ -17,23 +17,91 @@ original Secret from the SealedSecret.
 See https://github.com/bitnami-labs/sealed-secrets/releases for the latest
 release.
 
-```sh
+
+### GKE Requirements
+
+If installing on a GKE cluster, a ClusterRoleBinding may be needed to successfully deploy the controller in the final command.  Replace <your-email> with a valid email, and then deploy the cluster role binding:
+ 
+```bash
+USER_EMAIL=<your-email>
+kubectl create clusterrolebinding $USER-cluster-admin-binding --clusterrole=cluster-admin --user=$USER_EMAIL
+```
+ 
+If you are using a **private GKE cluster**, you are required to create a Master-to-Node firewall rule to allow GKE to communicate to the kubeseal container endpoint port tcp/8080.
```

Comment:
uh, why is that needed? to get the proxy to work (and thus fetch the certificate?)

Reply:
Yes, when running kubeseal to get the public certificate I had to add a firewall rule to make this work. This is actually related to [this](https://linkerd.io/2/reference/cluster-configuration/#private-clusters) which is a common issue in GKE private clusters.

Reply:
I'm considering changing the way we advertize the pubkey.
Currently the controller serves the certificate and the client uses the api server proxy feature to talk directly with the controller.

I wanted to change this so that the certificate gets written into a resource (e.g. a config-map) which would be made readable to all authenticated users. `kubeseal` would then have only to talk to the API server.

I'm not against documenting a useful tip&trick for GKE, but I'd rather solve this in a less invasive way.

That said, in the meantime would the offline signing with `--cert` work for you?

Reply:
@mkmik that would be a good idea and it's totally fine for me. 👍 

I just wanted future GKE users of kubeseal to be aware of the issue. Because without the firewall settings, kubeseal output just displays a go runtime deadline exceeded error. What would be useful is to show a connection timeout error from `net.DialTimeout` or something.